src/styles/_globals.scss

```diff
@@ -23,17 +25,28 @@
   text-align: center;
 }
 
+img {
+  display: block;
+}
+
 body {
   margin: 0;
 }
 
+ul,
+li {
+  list-style-type: inherit;
```

Comment:
we can set **list-style-type: none** here, as we only need to reset these styles for lists and list items